include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -147,10 +145,14 @@ SpatialOperatorBase<dim, Number>::fill_matrix_free_data(
   matrix_free_data.insert_constraint(&constraint_u_scalar, field + dof_index_u_scalar);
 
   // quadrature
-  matrix_free_data.insert_quadrature(dealii::QGauss<1>(param.degree_u + 1), field + quad_index_u);
-  matrix_free_data.insert_quadrature(dealii::QGauss<1>(param.get_degree_p(param.degree_u) + 1),
+  matrix_free_data.insert_quadrature(fe_u->reference_cell().template get_gauss_type_quadrature<dim>(
```

Comment:
I think it would be good to have one common style in ExaDG, see for comparison the `Poisson` module

https://github.com/exadg/exadg/blob/9aba12677bb4025a2608eeff5b5f0c396932527c/include/exadg/poisson/spatial_discretization/operator.cpp#L235-L241

and the `Structure` module

https://github.com/exadg/exadg/blob/9aba12677bb4025a2608eeff5b5f0c396932527c/include/exadg/structure/spatial_discretization/operator.cpp#L317-L323